File: src/ApiHub/Api.Hub.Domain/DTOs/GameConfigDto.cs
using Api.Hub.Domain.GameConfig;

namespace Api.Hub.Domain.DTOs
{
    public class GameConfigDto
    {
        public int WorldWidth { get; private set; }
        public int WorldHeight { get; private set; }
        public int InitialRadius { get; private set; }
        public string RegisteredName { get; private set; }

        public GameConfigDto(string name)
        {
            WorldWidth = CanvasConfig.WorldWidth;
            WorldHeight = CanvasConfig.WorldHeight;
            InitialRadius = BubbleConfig.InitialPlayerRadius;
            RegisteredName = name;
        }

        public GameConfigDto(){ }
    }
}

File: src/spa/src/app/game/multiplayer/GameConfigDto.ts
export class GameConfigDto {
  public WorldWidth: number;
  public WorldHeight: number;
  public InitialRadius: number;
  public RegisteredName: string;

  constructor(width: number, height: number, x: number, y: number, r: number, name: string) {
    this.WorldHeight = height;
    this.WorldWidth = width;
    this.InitialRadius = r;
    this.RegisteredName = name;
  }
}
